.yarnrc.yml

```diff
@@ -0,0 +1,2 @@
+enableGlobalCache: false
+nodeLinker: node-modules
```

Comment:
As recommended in https://github.com/eirslett/frontend-maven-plugin/blob/4a501a10677716a0e6676b603e3b93bc6bf319f1/frontend-maven-plugin/src/it/corepack-provided-integration/.yarnrc.yml. Note that without the `nodeLinker` line https://github.com/eirslett/frontend-maven-plugin/pull/1157 does not actually work, instead complaining about `corepack` not being listed in `package.json`. I filed for another day any latent curiosity about why this might be the case (probably a deficiency  in `frontend-maven-plugin` documentation at the very least).

Reply:
why not using 'pnp' https://yarnpkg.com/migration/pnp

Reply:
I see no reason why using PNP would solve the problem statement described in this PR description.